Explain that you can't install Mu on 32-bit systems (#64)
mu-editor/mu-editor.github.io: en/howto/1.1/install_with_python.md

        Python 3 installed and that when Python was installed, you clicked the
        option that asks you if you'd like Python added to your path (you
        do).</li>
        <li>You have a 32-bit system. Some of Mu's dependencies will not run on a
        32-bit system, so you will need to upgrade your system or use a different

>         editor.</li>

Reviewer: Thanks for the PR @ajs256!

I this case the 32-bit restriction is only for Linux, as I believe there are 32 bit wheels for Windows available. So we should clarify that this entry.

>         on a 32-bit system, so you will need to upgrade your system or use a 
>         different editor.</li>

Reviewer: ```suggestion
        on a 32-bit system, so you will need to upgrade your system to be able
        to install Mu.
```
